react/features/base/tracks/actions.native.ts

```diff
@@ -63,6 +63,7 @@ async function _startScreenSharing(dispatch: IStore['dispatch'], state: IReduxSt
             }, NOTIFICATION_TIMEOUT_TYPE.LONG));
         }
     } catch (error: any) {
-        console.log('ERROR creating screen-sharing stream ', error);
+        console.error('ERROR creating screen-sharing stream ', error);
```

Comment:
This should be using the logger indeed.

Also the ERROR is redundant since it's already implied by the log level.